tests/test_logic.py

```diff
@@ -159,6 +160,7 @@ def test_complete_task_updates_workflow_state(empty_workflow_state):
 def test_complete_task_includes_created_time(empty_workflow_state):
     task_id = "Task1"
     now = datetime.now(timezone.utc)
+    sleep(0.1)
```

Comment:
Did you have some problems with the test? I think it runs fine locally and with github actions.

Reply:
Without this addition `test_complete_task_includes_created_time` fails on my local as variables `now` and `actual_created` match exactly. The assertion in this test requires `now` to be less and not equal to `actual_created`. I opted for `sleep` as changing that assertion to accept equal values might be against the intent of this test.

Reply:
Ok, I see. That's strage it came up so late in the process. I don't think using sleep here is a good idea though.
How about transform the `now` variable to `second_ago` with help of timedelta
```
second_ago = datetime.now() - datetime.timedelta(seconds=1)
```

Reply:
I've made the recommended change in commit 14501ab49cbf42d17c87a97e445a5034e3949c30